core/templates/App.ts

```diff
@@ -284,27 +284,28 @@ angular.module('oppia').config([
 ]);
 
 angular.module('oppia').config(['$provide', function($provide) {
-  $provide.decorator('$log', ['$delegate', 'DEV_MODE',
-    function($delegate, DEV_MODE) {
-      var _originalError = $delegate.error;
+  $provide.decorator(
+    '$log', ['$delegate', 'DEV_MODE',
```

Comment:
```suggestion
  $provide.decorator('$log', [
      '$delegate', 'DEV_MODE', ...
```
This should be an accepted patter^, right? If yes, then I think we should follow this pattern!

This would work for `it`, `describe` & `angular.module('oppia')....` functions, we don't have to count them as exception*

/cc @bansalnitish

Reply:
It is not necessary that this pattern will be followed every where, what if there are too many arguments before the square bracket then we can not be able to what you suggested, the same case for other examples you have given. right?

FYI, I've allowed to have `[`, `{` at the end of any line

Reply:
> what if there are too many arguments before the square bracket then we can not be able to what you suggested, the same case for other examples you have given. right?

If there are too many arguments before square bracket and they can fit on one line then you can still break after `[` but fi the two may args before `[` are in multiline then you have to do the break after `(` of the function call!

> the same case for other examples you have given. right?

Can you please explain this?

Reply:
> If there are too many arguments before square bracket and they can fit on one line then you can still break after [ but fi the two may args before [ are in multiline then you have to do the break after ( of the function call!

The check I implemented allows the pattern you are suggesting, there will be no error on your suggested code.